docs/changehistory/NextVersion.md

```diff
@@ -27,3 +28,27 @@ Some detail and requirements are as following.
 ### Calculated properties specification enhancements
 
 A new optional [`extendedData`]($docs/presentation/content/CalculatedPropertiesSpecification.md#attribute-extendeddata) attribute has been added to [calculated properties specification]($docs/presentation/content/CalculatedPropertiesSpecification.md). The attribute allows associating resulting calculated properties field with some extra information, which may be especially useful for dynamically created calculated properties.
+
+### Quantity
+
+- Add support for 'Ratio' format type (e.g. "1:2")
+  - Example: Formatting a Ratio
+  - Assuming that a `UnitsProvider` has been registered and initialized, here's how to format a ratio:
+```ts
+const ratioFormatProps: FormatProps = {
+    type: "Ratio",
+    ratioType: "OneToN",  // Formats the ratio in "1:N" form
+    composite: {
+        includeZero: true,
+        units: [
+            { name: "Units.HORIZONTAL_PER_VERTICAL" },
+        ],
+    },
+};
+
+const ratioFormat = new Format("Ratio");
+ratioFormat.fromJSON(unitsProvider, ratioFormatProps).catch(() => {});
+```
+
+- Add support for unit inversion during unit conversion
```

Comment:
do we have any further documentation explaining how to enable unit inversion? I remember going through the code that there are different enums for UnitConversionInversion. Perhaps writing down the enums available and explaining what they do will be helpful.

Reply:
I added more description to the enum "UnitConversionInvert" values explaining their purpose.